bbcode-src/tags/rowcolumn.js

```diff
@@ -0,0 +1,16 @@
+import { preprocessAttr, toNode } from "../utils/common";
+
+/**
+ * @file Adds [row][column] to bbcode
+ * @example Adds [row][column][/column][/row]
+ */
+export const rowcolumn = {
+  row: (node) => toNode("div", { class: "bb-row" }, node.content),
+  column: (node) => {
+    const columnAttrs = preprocessAttr(node.attrs)._default || "1";
```

Comment:
[pebble] It looks like the default isn't handled on the live site. So  `[column]test[/column]` breaks the tag and `[column=span]test[/column]` will return a column with no matching span styling. I am fine with the default being a span of 1, but I wonder if it would make more sense to be the full width instead? (so span8) That way, by default, a non-specified column will take up max width?

As noted, this is a non-blocking pebble comment; just curious of both of your thoughts.  @blythechan @Alteras1

Reply:
span 8 would make more sense.

Reply:
Ok, will do.